100-Appendix/025-sla.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Xata Service Level Agreement (SLA)
+navTitle: SLA
+keywords: ['sla', 'slo', 'service level agreement']
```

Comment:
```suggestion
keywords: ['sla', 'slo']
```

Keywords are heavily weighted in search. I don't think we want this showing for a generic term like "service" above technical docs.